src/saveDocs.js

```diff
@@ -4,45 +4,62 @@ import { getDownloads, getDependents } from './npm.js';
 import { getChangelogs } from './changelog.js';
 import { getHits } from './jsDelivr.js';
 import { getTSSupport } from './typescriptSupport.js';
+import datadog from './datadog.js';
+
+export default async function saveDocs({ docs, index }) {
+  const start = Date.now();
 
-export default function saveDocs({ docs, index }) {
   const rawPkgs = docs
     .filter(result => result.doc.name !== undefined) // must be a document
-    .map(result => formatPkg(result.doc))
+    .map(result => {
+      const start1 = Date.now();
+
+      const formatted = formatPkg(result.doc);
+
+      datadog.timing('formatPkg', Date.now() - start1);
+      return formatted;
+    })
     .filter(pkg => pkg !== undefined);
 
   if (rawPkgs.length === 0) {
     log.info('🔍 No pkgs found in response.');
     return Promise.resolve();
   }
 
-  return addMetaData(rawPkgs)
-    .then(pkgs => index.saveObjects(pkgs))
-    .then(() => log.info('🔍 Found and saved %d packages', rawPkgs.length));
+  let start2 = Date.now();
+  const pkgs = await addMetaData(rawPkgs);
+  datadog.timing('saveDocs.addMetaData', Date.now() - start2);
+
+  start2 = Date.now();
+  index.saveObjects(pkgs);
+  datadog.timing('saveDocs.saveObjects', Date.now() - start2);
+
```

Comment:
the log "found and saved N packages is gone here now, is it now in index.js?

Reply:
yes, I wanted to have the maximum of logs at same place. 
I should have put a screenshot, but the log looks like that right now for better observability (all naming and log are debatable ☺️ )

![Screenshot 2019-08-03 14 56 46](https://user-images.githubusercontent.com/1637651/62412175-11b6ee80-b5ff-11e9-8f53-97063ab10049.png)